org.eclipse.scanning.test/src/org/eclipse/scanning/test/points/CompoundTest.java

```diff
@@ -126,28 +126,28 @@ public void testSimpleCompoundStep2Step() throws Exception {
 		final List<IPosition> points = scan.createPoints();
 
 		// 290K
-		assertEquals(new Double(290), (Double)points.get(0).get("Temperature"));
-		assertEquals(new Double(1),   (Double)points.get(0).get("Position"));
-		assertEquals(new Double(290), (Double)points.get(1).get("Temperature"));
-		assertEquals(new Double(1.6), (Double)points.get(1).get("Position"));
-		assertEquals(new Double(290), (Double)points.get(2).get("Temperature"));
-		assertEquals(new Double(2.2), (Double)points.get(2).get("Position"));
+		assertEquals(new Double(290), points.get(0).get("Temperature"));
+		assertEquals(new Double(1),   points.get(0).get("Position"));
+		assertEquals(new Double(290), points.get(1).get("Temperature"));
+		assertEquals(new Double(1.6), points.get(1).get("Position"));
+		assertEquals(new Double(290), points.get(2).get("Temperature"));
+		assertEquals(new Double(2.2), points.get(2).get("Position"));
 
 		// 291K
-		assertEquals(new Double(291), (Double)points.get(6).get("Temperature"));
-		assertEquals(new Double(1),   (Double)points.get(6).get("Position"));
-		assertEquals(new Double(291), (Double)points.get(7).get("Temperature"));
-		assertEquals(new Double(1.6), (Double)points.get(7).get("Position"));
-		assertEquals(new Double(291), (Double)points.get(8).get("Temperature"));
-		assertEquals(new Double(2.2), (Double)points.get(8).get("Position"));
+		assertEquals(new Double(291), points.get(6).get("Temperature"));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this "Double" constructor [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2129)